src/main/java/gov/cms/madie/models/measure/Measure.java

```diff
@@ -106,6 +106,8 @@ public class Measure extends ResourceAcl {
   private String lastModifiedBy;
   private Date measurementPeriodStart;
   private Date measurementPeriodEnd;
+  private List<SupplementalData> supplementalData;
+  private List<RiskAdjustment> riskAdjustments;
```

Comment:
for some reason, I remember @sb-cecilialiu also working on a similar thing.  
Also, I don't think we need two different models for this. they are both supplemental data elements with small distinction. I was thinking if we could do this

List<SepplimentalDataElement> sepplimentalDataElements;
where SepplimentalDataElement is,
SepplimentalDataElement {
id,
cqlDefinition: string
description: string
type: Enum(supplemental-data | risk-adjustment-factor)
}  

what do you think? cc:@jkotanchik-SB , @RohitKandimalla

Reply:
I believe it will be the same for QDM as well, so I don't have anything against having a single structure. LGTM

Reply:
I'm fond of readability when all other factors being equal. Unless we're gaining a decent benefit from combining the two, I'd prefer they stay separate.

Based on what @adongare said about their relationship, I'm thinking RAV could be nearly empty subclass of SDE to capture that relationship.

Reply:
Okay, separate models then.